src/detect-engine.c

```diff
@@ -1427,6 +1427,19 @@ void InspectionBufferInit(InspectionBuffer *buffer, uint32_t initial_size)
     }
 }
 
+/** \brief setup the buffer empty */
+void InspectionBufferSetupMultiEmpty(InspectionBuffer *buffer)
```

Comment:
A little late, but more of a general thing.  We should think about prefixing all "public" functions going forward.